src/recompress.rs

```diff
@@ -55,7 +55,7 @@ pub(crate) fn recompress_file(
         let mut lzma_ops = xz2::stream::LzmaOptions::new_preset(9).unwrap();
         // This sets the overall dictionary size, which is also how much memory (baseline)
         // is needed for decompression.
-        lzma_ops.dict_size(64 * 1024 * 1024);
+        lzma_ops.dict_size(128 * 1024 * 1024);
```

Comment:
I think you mentioned on Zulip that setting this in excess of the file size being compressed is wasteful (or maybe I misinterpreted and it's just useless?) -- should we tweak that as well?